src/Perpetual.ts

```diff
@@ -92,6 +93,38 @@ export class Perpetual {
     this.contracts.setProvider(provider, networkId);
   }
 
+  public setDefaultAccount(
+    account: address,
+  ): void {
+    this.web3.eth.defaultAccount = account;
+    this.contracts.setDefaultAccount(account);
+  }
+
+  public getDefaultAccount(): address {
+    return this.web3.eth.defaultAccount;
+  }
+
+  public loadAccount(
```

Comment:
can we also replicate the thing where you can pass in accounts to load in `PerpetualOptions` the same way we do for solo?